dev/build.image/profiles/microProfile7/features.xml

```diff
@@ -1,2 +1,15 @@
 <feature>microProfile-7.0</feature>
 <feature>localConnector-1.0</feature>
+<feature>cdi</feature>
+<feature>jsonb</feature>
+<feature>jsonp</feature>
+<feature>mpJwt</feature>
+<feature>mpTelemetry</feature>
+<feature>mpFaultTolerance</feature>
+<feature>mpOpenAPI</feature>
+<feature>mpRestClient</feature>
+<feature>mpHealth</feature>
+<feature>mpMetrics</feature>
+<feature>mpConfig</feature>
```

Comment:
```suggestion
<feature>mpConfig</feature>
<feature>mpFaultTolerance</feature>
<feature>mpHealth</feature>
<feature>mpJwt</feature>
<feature>mpMetrics</feature>
<feature>mpOpenAPI</feature>
<feature>mpRestClient</feature>
<feature>mpTelemetry</feature>
```

If we are adding the `mpMetrics` versionless feature, we really should include the `mpMetrics-5.1` feature to the profile as well.  It will come for free, but it should still be added.